pkg/controllers/kube-apiserver.go

```diff
@@ -125,7 +125,7 @@ func (s *KubeAPIServer) configure(cfg *config.Config) error {
 			"kubelet-certificate-authority":   {cryptomaterial.CABundlePath(kubeCSRSignerDir)},
 			"kubelet-client-certificate":      {cryptomaterial.ClientCertPath(kubeletClientDir)},
 			"kubelet-client-key":              {cryptomaterial.ClientKeyPath(kubeletClientDir)},
-			"kubelet-preferred-address-types": {"InternalIP", "Hostname"},
+			"kubelet-preferred-address-types": {"Hostname", "InternalIP"},
```

Comment:
A comment here (or in the commit message) explaining how this change fixes the problem would help future maintainers.

Reply:
Done